provider/generic_keycloak_identity_provider.go

```diff
@@ -89,12 +99,72 @@ func resourceKeycloakIdentityProvider() *schema.Resource {
 				Default:     "",
 				Description: "Alias of authentication flow, which is triggered after each login with this identity provider. Useful if you want additional verification of each user authenticated with this identity provider (for example OTP). Leave this empty if you don't want any additional authenticators to be triggered after login with this identity provider. Also note, that authenticator implementations must assume that user is already set in ClientSession as identity provider already set it.",
 			},
+			// all schema values below this point will be configuration values that are shared among all identity providers
+			"extra_config": {
```

Comment:
So basically extra config(and gui_order and sync_mode) was moved to the generic identity provider, :+1: